package.json

```diff
@@ -48,6 +48,6 @@
     "webpack-dev-server": "^3.11.0"
   },
   "peerDependencies": {
-    "cypress": "^2.1.0 || ^3.1.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0"
+    "cypress": "^2.1 || ^3.1 || 4 - 13"
```

Comment:
I like the version range annotation. Is it also possible to do `2 - 13`? Or is it necessary that `2.1` and `3.1` are listed separately?

Reply:
I think the explicit support for `2.1.x` and `3.1.x` was introduced by accident a long time ago. IMO having `2 - 13` is much better. Thank you for pointing that out 😃.

Reply:
Upon closer consideration, perhaps it is time to start dropping support for these older versions? They are not actually being tested against on CI, which makes it highly doubtful that this library is actually still compatible with these older versions.

Then again I could be wrong about that, I have no idea how much (or little) the API for Cypress has changed. Perhaps this is also better served as a separate issue/discussion.

Reply:
Great, updated the version range as discussed :+1: